charts/openmetadata/templates/check-db-migrations-job-hook.yaml

```diff
@@ -7,6 +7,13 @@ metadata:
     "helm.sh/hook": pre-upgrade 
     "helm.sh/hook-weight": "-1" # we use a smaller weight so it's created before the job  
     "helm.sh/hook-delete-policy": hook-succeeded, hook-failed
+    {{- if .Values.annotations }}
+    {{- range $key, $val := fromYaml .Values.annotations }}
```

Comment:
Remove `fromYaml` from this condition. Helm charts installation and upgrades are failing due to this!

Reply:
Removed fromYaml from this condition